src/components/pages/creative/controller/components/ImageFilters.js

```diff
@@ -16,7 +16,6 @@ const ImageFilters = () => {
         {filterObject: new fabric.Image.filters.Technicolor(), filterName: 'Technicolor'},
         {filterObject: new fabric.Image.filters.Invert(), filterName: 'Invert'}
     ]
-
     const handleFilter = (name) => {
         const FO = filters.find(f => f.filterName === name)
```

Comment:
**Issue**: The 'find' method is used without proper null handling for 'filterName'. If 'filterName' is not found, 'FO' will be undefined, and accessing properties on it will cause runtime errors. <br> **Fix**: Add a check to ensure 'FO' is not undefined before proceeding with operations on it. <br> **Code Suggestion**: 
 ```
 -        if(!FO) return
 +        if(!FO) {
 +          console.error('Filter not found');
 +          return;
 +        }
 ```